scheduler/src/cook/config.clj

```diff
@@ -509,6 +509,8 @@
                                                         {:json-value (str lock-objects)})
                              :default-workdir "/mnt/sandbox"
                              :max-jobs-for-autoscaling 1000
+                             :autoscale-blocklist-seconds 240 ; Should be greater than the time to start a pod and have it accept workloads.
```

Comment:
Ok.

Reply:
Thoughts on avoiding the prep-then-throwaway problem?

Reply:
Too awkward to do in the code. Only the CC knows if it is running there (or is starting), so it makes sense for it to do the filtering. A better fix would be to make prep cheaper, or increase the timeout. 

Assuming no races and instantaneous operations, we could have a timeout of infinity; the reason to have a finite timeout is because of those races.